slackbot/utils.py

```diff
@@ -10,6 +10,11 @@
 
 logger = logging.getLogger(__name__)
 
+till_white = "([^\s]+)"
+till_end = "(.*)"
+issue = "(\\S[^\\-\\s]*-[*\\d])"
+j_key = "(\\S[^\\-\\s]*)"
```

Comment:
are issue and j_key JIRA-related? if so, kill them here and we'll include this in the JIRA plugin